OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -20,46 +20,157 @@ namespace OpenRA.Mods.Common.Widgets
 	public class SupportPowerTimerWidget : Widget
 	{
 		public readonly string Font = "Bold";
-		public readonly string Format = "{0}: {1}";
-		public readonly TimerOrder Order = TimerOrder.Descending;
+		public readonly string TimerFormat = "{0}: {1}";
```

Comment:
That would break FieldLoader's ability to override the default with values defined in the yaml.